src/pages/profiles/ProfileList.tsx

```diff
@@ -56,62 +129,122 @@ const ProfileList: FC = () => {
           "aria-label": "Name",
         },
         {
-          content: profile.description,
+          content: (
+            <div className="u-truncate" title={profile.description}>
+              {profile.description}
+            </div>
+          ),
           role: "rowheader",
           "aria-label": "Description",
         },
         {
-          content: getProfileInstances(
-            project,
-            isDefaultProject,
-            profile.used_by
-          ).length,
+          content: (
+            <>
+              {isDefaultProject && (
+                <>
+                  {total} in all projects
+                  <br />
+                </>
+              )}
+              {usedBy} instances
+            </>
+          ),
           role: "rowheader",
-          className: "u-align--right",
+          className: "u-text--muted",
           "aria-label": "Used by",
         },
       ],
       sortData: {
         name: profile.name.toLowerCase(),
         description: profile.description.toLowerCase(),
-        used_by: profile.used_by,
+        used_by: usedBy,
       },
     };
   });
 
+  const pagination = usePagination(rows);
+
+  useEventListener("resize", () => updateTBodyHeight("profile-table-wrapper"));
+  useEffect(() => {
+    updateTBodyHeight("profile-table-wrapper");
+  }, [
+    profiles,
+    notify.notification,
+    query,
+    pagination.pageSize,
+    pagination.currentPage,
+  ]);
+
   return (
-    <>
-      <BaseLayout
-        title="Profiles"
-        controls={
+    <main className="l-main profile-list">
+      <div
+        className={classnames("p-panel", {
+          "has-side-panel": !!panelParams.profile,
+        })}
+      >
+        <div className="p-panel__header profile-list-header">
+          <h1 className="p-heading--4 u-no-margin--bottom">Profiles</h1>
+          <SearchBox
```

Comment:
Should the search be left aligned like in the instance list?
The small input looks a little lost on large widths atm.

Reply:
This is a question for @al-bakalova - currently in the design it doesn't seem to be left-aligned like in the instance list.

Reply:
Good catch!
We should probably left align it to match with the Instance table and that way it won't jump when we switch the pages.

Reply:
Ok good, I will update it accordingly. And this will also solve the issue mentioned in [this other comment](https://github.com/canonical/lxd-ui/pull/298#discussion_r1171084327).